Algorithmic/sort_three_ints/sort_three_ints.rs

```diff
@@ -0,0 +1,49 @@
+use std::io;
+use std::cmp;
+
+fn main() {
+    let mut a = String::new();
+    let mut b = String::new();
+    let mut c = String::new();
+    
+    println!("Enter a value for a.");
+    io::stdin().read_line(&mut a)
+        .expect("Failed to read line.");
+
+    println!("Enter a value for b.");
+    io::stdin().read_line(&mut b)
+        .expect("Failed to read line.");
+
+    println!("Enter a value for c.");
+    io::stdin().read_line(&mut c)
+        .expect("Failed to read line.");
+
+    let a: i32 = match a.trim().parse() {
+        Ok(num) => num,
+        Err(_) => 0
+    };
+
+    let b: i32 = match b.trim().parse() {
+        Ok(num) => num,
+        Err(_) => 0
+    };
+
+    let c: i32 = match c.trim().parse() {
+        Ok(num) => num,
+        Err(_) => 0
+    };
+    
+    // To computate the max of two numbers the cmp::max() function is used.
+    // Since it only takes in 2 numbers at a time, first, we find the 
```

Comment:
Ooh, the max function is very useful for beginner Rust devs! I need to take a look at this more myself, and see if I can tidy up any of my code with max instead of my usual max implementation xD

Reply:
Gotta use Google more often :D